src/syscall/syscall_libc.go

```diff
@@ -60,6 +61,5 @@ func splitSlice(p []byte) (buf *byte, len uintptr) {
 	return slice.buf, slice.len
 }
 
-// ssize_t write(int fd, const void *buf, size_t count)
 //export write
-func libc_write(fd int32, buf *byte, count uint) int
+func libc_write(fd int32, buf *byte, size uint) int
```

Comment:
There are still a few changes in this file that look unintentional. They're not necessarily bad, but probably shouldn't be in this PR.

Reply:
@racerxdl this one.

Reply:
Done!